openwisp_controller/geo/tests/pytest.py

```diff
@@ -108,7 +104,3 @@ async def test_consumer_staff_but_no_change_permission(self):
         connected, _ = await communicator.connect()
         assert connected
         await communicator.disconnect()
-
-    def test_routing(self):
-        application = import_string(getattr(settings, 'ASGI_APPLICATION'))
-        assert isinstance(application, ProtocolTypeRouter)
```

Comment:
can this test be updated to assert the expected type matches?

Reply:
I removed this test because it seemed redundant now that WebSocketConsumer is using the project's ASGI application.

Reply:
I have restored and updated the test.